### ISSUE TITLE
Accessibility for VoiceOver users

### ISSUE THREAD

**Issue by mathewa6** [xhacker/TEAChart#19]
### Issue
First, thanks for the great library! I'm opening this issue since it's a bit more code than can comfortably fit in a pull request. Currently all 3 charts : Bar, Contribution and Clock are not visible to blind users or anyone using the VoiceOver utility in iOS (Settings > General > Accessibility > VoiceOver). 

### Details/Reproduce
When used in an app, such as the Demo project: Bar and Clock charts are **ignored** (as can be verified using the Xcode Accessibility Inspector in Developer Tools) and Contribution chart outputs unusable information since it defaults to each block acting as a button.

### Fix/Proposal
I have fixed this issue by implementing the UIAccessibility methods defined [here](https://developer.apple.com/reference/uikit/uiaccessibility). Since the branch I modified these on edited the implementation files of all 3 classes and the README, I'm opening this issue to both bring your attention to the feature as well as suggest that I open a pull request that fixes the above. 

**Note** that the accessibility feature in no way affects regular use of TEAChart in any way. These methods simply allow users who rely on VoiceOver to now use the data contained in the chart.

**Comment by anti-transmission** [comment]
Thank you for your effort to make the library accessible! We look forward to your pull request. 😊

**Comment by mathewa6** [comment]
Thank you! I've submitted PR #20 .
There are abundant comments so users can follow what each line does. I can remove these if preferred as well.
